src/layouts/base.njk

```diff
@@ -5,22 +5,29 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>{{title}}</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Faustina:ital,wght@0,600;0,700;1,400&family=Montserrat:wght@100;200;300;400;500;600;700;800;900&family=Raleway:wght@300;400&family=Signika+Negative:wght@300;400;600&display=swap" rel="stylesheet">
```

Comment:
Are you using all of these font weights? Especially for Montserrat, that's a whole lotta font to download. It's not the biggest performance issue if you check Lighthouse, but it would be quick win to pare down the fonts a little.

Reply:
Honestly, no probably not. I bet I just grabbed them all, thinking that was a good idea at the time. Fixed now. https://github.com/marissahuysentruyt/marissahuysentruyt/pull/21/commits/786c7ca73fc62eb5b720f15ca9d445389e0b1da2